Engine/media/audio/audio.cpp

```diff
@@ -428,39 +386,6 @@ ScriptAudioChannel* play_audio_clip_by_index(int audioClipIndex)
         return NULL;
 }
 
-void stop_and_destroy_channel_ex(int chid, bool resetLegacyMusicSettings) {
```

Comment:
Could these be moved back so we can compare changes?

Reply:
It also makes merging much easier if you don't move code around. :)

Reply:
I'll file the appropriate paperwork to move it back for purposes of analyzing this PR but frankly I don't care if it makes merging harder. If your workflow means you can't move code, then there's a problem with your workflow. Mixing it with other commits was stupid though. I forgot I'd done that early on before I had a better plan.

Reply:
Come on, this is a collaborative project, we care if it makes it harder for others to merge.

Reply:
> there's a problem with your workflow

Yes, there is a problem with workflow: it is that in this branch we have to support multiple legacy versions of ags. Since we not always have an actual specification telling how these versions should work, every time someone finds a regression I have to go back in time and compare code.